libs/components/phone-field/src/lib/modules/phone-field/phone-field.component.scss

```diff
@@ -1,6 +1,6 @@
 @use 'libs/components/theme/src/lib/styles/mixins' as mixins;
 @use 'libs/components/theme/src/lib/styles/variables' as *;
-@use 'node_modules/intl-tel-input/build/css/intlTelInput';
+@import 'node_modules/intl-tel-input/build/css/intlTelInput.css';
```

Comment:
Switching to `@import` to load this file as CSS (not SASS), which fixes an issue with `esbuild` and resolving the background image `url()`.